licenses.d2iq.yaml

```diff
@@ -1,8 +1,10 @@
 ignore:
-  - ghcr.io/mesosphere/git-operator:v0.2.0
+  - docker.io/mesosphere/git-operator:v0.8.3
+  - docker.io/mesosphere/gitwebserver:v0.8.3
```

Comment:
Can you add mappings for these?

Reply:
Thanks, completely forgot that we do have mappings for components we build as well.